docs/tutorials/first-experiment.md

```diff
@@ -179,15 +174,14 @@ To run the experiment locally, we can run the script directly:
 
 === "CPU"
     ```bash
-    python experiments/tutorial/train_tiny_model_cpu.py --prefix /tmp/marin-tutorial
+    python experiments/tutorial/train_tiny_model_cpu.py --prefix local_store
```

Comment:
I don't like storing stuff in `/tmp` because it's easy to just forget about and not cleanup.

Reply:
@but it's self cleaning! (i'm fine with whatever)